Shared/Game.Shared.External/DependencyInjection.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Game.Shared.External.Providers.Ai;
+using Game.Shared.External.Providers.Ai.Llama;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Game.Shared.External
+{
+    /// <summary>
+    /// Extensions for the dependency injection container.
+    /// </summary>
+    public static class DependencyInjection
+    {
+        /// <summary>
+        /// Adds the shared external modules into the dependency injection container.
+        /// </summary>
+        public static IServiceCollection AddAiProvider(this IServiceCollection services)
+        {
+            services.AddSingleton<IAiProvider, LlamaAiProvider>();
```

Comment:
To nie powinno być generyczne, `AddAIProvider<T>`, żeby można było dość dynamicznie podmienić z `LlamaAiProvider` na coś innego (np. pomiędzy produkcją i developmentem)?